pa_matchbatch.sh

```diff
@@ -78,8 +82,8 @@ getComponentRevMap() {
 # repo:fix_version, 1 per line
 #
 getFixVerFor() {
-	if [[ -f  ${OVERRIDE_PATH} ]]; then
-		override=$(grep ${1}  ${OVERRIDE_PATH} | cut -d: -f2)
+	if [[ -f  ${ABSOLUTE_OVERRIDE_PATH} ]]; then
```

Comment:
Why not just use `realpath` on whatever the user passes in for the `OVERRIDE_PATH` ? It should noop for an absolute path.

Reply:
Somehow when I tested that it failed, but it seems to be working now. I've updated the PR.